framework/src/application/node/node.ts

```diff
@@ -474,6 +480,16 @@ export class Node {
 					params.data,
 					params.peerId,
 				),
+			getSchema: () => ({
+				account: this._chain.accountSchema,
+				baseBlockHeader: blockSchema,
+				blockHeaders: {
+					0: BlockProcessorV0.schema,
+					2: BlockProcessorV2.schema,
+				},
+				baseTransaction: BaseTransaction.BASE_SCHEMA,
+				transactions: this._getRegisteredTransactionSchemas(),
```

Comment:
```suggestion
				transactionsAssets: this._getRegisteredTransactionSchemas(),
```

Reply:
The request in the issue was to use transactions @shuse2 ?

Reply:
Would suggest updating the issue, as for both transactions and block headers, we can specify custom assets schemas not the full transaction and block schema.